.github/workflows/jobs.yml

```diff
@@ -252,6 +252,12 @@ jobs:
           make check
   windows_latest_cmake:
     runs-on: windows-latest
+    name: Windows-${{ matrix.msystem }}-INTERFACE64=${{ matrix.int64 }}
+    strategy:
+      fail-fast: false
+      matrix:
+        msystem: [ UCRT64, CLANG64]
```

Comment:
What's UCRT64/CLANG64? Can you detail a bit?

Reply:
https://www.msys2.org/docs/environments/